src/lib/UI/Dataset/CustomUrlsDataset.php

```diff
@@ -43,7 +50,24 @@ public function __construct(
      */
     public function load(Location $location): self
     {
-        $customUrlAliases = $this->urlAliasService->listLocationAliases($location, true, null, true);
+        try {
+            $customUrlAliases = $this->urlAliasService->listLocationAliases(
+                $location,
+                true,
+                null,
+                true
+            );
+        } catch (BadStateException $e) {
+            $this->logger->warning(
+                sprintf(
+                    'At least one custom alias belonging to location %d is broken. Fix it by using the ezplatform:urls:regenerate-aliases command.',
+                    $location->id
+                ),
+                [$e->getMessage()]
+            );
```

Comment:
Context is passed incorrectly.
```suggestion
            $this->logger->warning(
                sprintf(
                    'At least one custom alias belonging to location %d is broken. Fix it by using the ezplatform:urls:regenerate-aliases command.',
                    $location->id
                ),
                ['exception' => $e]
            );
```